schema.json

```diff
@@ -506,13 +594,19 @@
             "type": "string"
           }
         },
+        "required": [
+          "name",
+          "startDate",
+          "endDate"
```

Comment:
Not sure about `endDate` as not providing it means current as far as I thought. Please correct me, if I missed something.

Reply:
Sorry I will fix it.